frontend/src/metabase/dashboard/components/QuestionPicker/QuestionPicker.tsx

```diff
@@ -84,24 +84,24 @@ function QuestionPickerInner({
         onChange={handleSearchTextChange}
       />
 
-      {(hasDataAccess || hasNativeWrite) && !isEmbeddingSdk && (
```

Comment:
move the condition down to only the SQL button because we now want to show the `New Question` button on the SDK as well.